tools/c7n_left/README.md

```diff
@@ -9,15 +9,31 @@ source asset evaluation.
 
 ## Install
 
-We currently only support python 3.10 on mac and linux. We plan to
-expand support for additional operating systems and python versions
-over time.
-
+We currently only support python > 3.10 on mac and linux, to run on windows
+we recommend using our docker images.
 
 ```shell
 pip install c7n_left
 ```
 
+We also provide signed docker images. These images are built on top of chainguard's [wolfi linux
+distribution](https://www.chainguard.dev/unchained/introducing-wolfi-the-first-linux-un-distro) which
+is designed to be minimal, auditable, and secure.
+
+```shell
+docker pull c7n_left:dev
+```
+
+Images signatures can be verified using [cosign](https://github.com/sigstore/cosign)
+
+```
+set IMAGE (docker image inspect cloudcustodian/c7n-left:dev -f '{{index .RepoDigests 0}}')
```

Comment:
Think bash/zsh syntax will be a friendlier default here

```suggestion
export IMAGE=$(docker image inspect cloudcustodian/c7n-left:dev -f '{{index .RepoDigests 0}}')
```

Reply:
good call out, thanks